Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/MainActivityIntegrationTest.java

```diff
@@ -94,15 +95,25 @@ public class MainActivityIntegrationTest {
 	@Mock
 	private File internalMemoryPath;
 
-	@InjectMocks
 	private MainActivityPresenter presenter;
 
+	private CountingIdlingResource idlingResource;
+
 	@Rule
 	public ActivityTestRule<MainActivity> launchActivityRule = new ActivityTestRule<>(MainActivity.class);
 
 	@Rule
 	public ScreenshotOnFailRule screenshotOnFailRule = new ScreenshotOnFailRule();
 
+	@Before
+	public void init() {
+		MockitoAnnotations.initMocks(this);
+		idlingResource = new CountingIdlingResource("name");
```

Comment:
Not fully certain why you just don't use the mainactity.getIdlingResource() and rather create a new one.